src/main/java/com/mongodb/jdbc/MongoSQLDatabaseMetaData.java

```diff
@@ -50,6 +50,13 @@
 @AutoLoggable
 public class MongoSQLDatabaseMetaData extends MongoDatabaseMetaData implements DatabaseMetaData {
 
+    private static final BsonBoolean BSON_FALSE_VALUE = new BsonBoolean(false);
+    private static final BsonInt32 BSON_ZERO_VALUE = new BsonInt32(0);
+    private static final BsonNull BSON_NULL_VALUE = new BsonNull();
+    private static final BsonString BSON_EMPTY_VALUE = new BsonString("");
```

Comment:
```suggestion
    private static final BsonString BSON_EMPTY_STR_VALUE = new BsonString("");
```
or
```
    private static final BsonString BSON_EMPTY_STRING_VALUE = new BsonString("");
```
[nit] I think the name "BSON_EMPTY_VALUE" is a bit misleading! Knowing the type helps a lot, but when reading through the code below, there isn't any visual indication of the type. I know it is a bit of an antipattern to include type info in variable names, but in this case it will help disambiguate what it represents. (For example, I could also name the empty array _or_ empty document constant `BSON_EMPTY_VALUE`, and there would be no way of knowing which type I'm referring to without going back to the declaration!)